### PR TITLE
chore: [AMR-593] Update docs to use new AMR resource name

### PR DESCRIPTION
**Pull request by xtreme-jason-smith**

### CODE REVIEW

File: scst-store/deployment-details.hbs.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The installation creates the following in your Kubernetes cluster:
     - `metadata-store-app`
     - `metadata-store-db`
     - `amr-cloudevent-handler`. If AMR is deployed, see [Deploying AMR](#amr).
-    - `artifact-metadata-repository-app`. If AMR is deployed, see [Deploying AMR](#amr).
+    - `amr-graphql-app`. If AMR is deployed, see [Deploying AMR](#amr).
```
Comment (xtreme-debbie-chen) [inline scst-store/deployment-details.hbs.md:18]: Isn't AMR deployed by default now? Shouldn't this note go away? I believe it now makes references to value fields that no longer exist. Or is this meant to be handled in another later story?

Comment (xtreme-jason-smith) [reply scst-store/deployment-details.hbs.md:18]: Technically that should be handled in a separate story. I'll coordinate with Tony to make sure the docs reflect that.

Comment (xtreme-jason-smith) [reply scst-store/deployment-details.hbs.md:18]: A new story has been created to address this issue.

**Review by xtreme-debbie-chen — APPROVED** [review]

**Review by shaileshpandey — APPROVED** [review]
LGTM

**Review by Manifaust — APPROVED** [review]

**Pull request closed without merging**